src/native/libs/System.Security.Cryptography.Native/opensslshim.h

```diff
@@ -144,6 +144,12 @@ void SSL_CTX_set_alpn_select_cb(SSL_CTX* ctx,
 void SSL_get0_alpn_selected(const SSL* ssl, const unsigned char** protocol, unsigned int* len);
 #endif
 
+
+#if OPENSSL_VERSION_NUMBER < OPENSSL_VERSION_1_1_1_RTM
```

Comment:
fine with me. But maybe there is standard way how to do it @bartonjs? There is osslcompat_111.h but that does not seems to be included in source-build.

Reply:
It should be in an osslcompat_ file.

https://github.com/dotnet/runtime/blob/d43c048b3698de191cbeccc3cda65787b4297aac/src/native/libs/System.Security.Cryptography.Native/opensslshim.h#L122-L130

If this is really only for "when building against the 1.1.0 headers", then it probably needs a special 1.1.0 section at the bottom, like

https://github.com/dotnet/runtime/blob/d43c048b3698de191cbeccc3cda65787b4297aac/src/native/libs/System.Security.Cryptography.Native/osslcompat_111.h#L83-L88

Reply:
Sorry, I have been staring at those osslcompat headers for half an hour and was not able to deduce in which one these should go, the `osslcompat_111.h` seems to contain 1.1.* function prototypes that are used when compiling against OpenSSL 1.0.*, but I need the `SSL_verify_client_post_handshake` etc. prototypes when compiling against 1.1.0 as well.

Functionally, `osslcompat_102.h` would probably do the trick, but that I understand that that file should contain only the prototypes for functions that were removed in 1.1.0....